features/install_from_list.feature

```diff
@@ -0,0 +1,70 @@
+Feature: Install from list
+
+Scenario: Install an app by getting the list of available apps for a mobile worker
+	Then I select "See Apps For My User (NEW!)" menu item
+	Then I verify that the current activity is "InstallFromListActivity"
+	
+	# Verify that we start out in mobile user auth mode
+	Then I see view with id "edit_domain"
+	Then I do not see view with id "edit_email"
+	
+	Then I press view with class "android.widget.Switch" 
+	
+	# Verify that we're now in web user auth mode
+	Then I do not see view with id "edit_domain"
+	Then I see view with id "edit_email"
+
+	# Switch back to mobile auth view
+	Then I press view with class "android.widget.Switch" 
+
+	# Test getting app list for a mobile user
+	Then I auth for install from list with username "test" and domain "commcare-tests" and password "123"
```

Comment:
Hey, not super comfortable with this being in code, we need to change this password and find a way to externalize